.github/workflows/confrimations-summary.yml

```diff
@@ -0,0 +1,29 @@
+name: Confirmations summary
+on:
+  push:
+    paths:
+      - ui/pages/confirmations/confirm/**
+  pull_request:
+    branches:
+      - develop
+    paths:
+      - ui/pages/confirmations/confirm/**
```

Comment:
Will we not already be notified of these changes as this will require a code review since we're code owners of `ui/pages/confirmations`?

Reply:
yeah, that was my question too when I started to work on this task, we just want to notify the devs already before review as soon as they open the PR